tests/mocks/mockInitDB.ts

```diff
@@ -0,0 +1,48 @@
+export default {
+	initManagers: function () {
+		class PostManager {
+			async getPost (postId: number): Promise<string> {
```

Comment:
I would definitely go with `jest.fn().mockResolvedValue()` or `jest.fn().mockImplementation()` to have better control in tests for all of the methods.

Reply:
Sure, I will replace them with sinon.stubs which are similar